internal/trace/context.go

```diff
@@ -48,6 +48,7 @@ func WithLoggerLevel(ctx context.Context, level logrus.Level) context.Context {
 
 	// create logger
 	logger := logrus.New()
+	formatter.DisableQuote = true
```

Comment:
What does it imply to disable quote?

Reply:
Because '\n' in the log is not interpreted as newline at the moment. It will be printed out as `\n` in the log. `formatter.DisableQuote = true` fixes this issue.

Reply:
And also, we are having logs such as 
`{\"verificationResults\":{\"SIGNATURE_VERIFIER.REVOCATION_CHECK\":{\"success\":true},\"SIGNATURE_VERIFIER.TRUSTED_IDENTITY\":{\"success\":true}},\"processedAttributes\":null}`

After disabling quote, it would be
`{"verificationResults":{"SIGNATURE_VERIFIER.REVOCATION_CHECK":{"success":true},"SIGNATURE_VERIFIER.TRUSTED_IDENTITY":{"success":true}},"processedAttributes":null}`

Reply:
Can you provide the full log? Otherwise, it makes no sense without context.

Reply:
The issue happens on Windows.
This is what I'm seeing without the flag setting to `true`:
```
 ./notation.exe verify testnotation.azurecr.io/tsa:v1 -d
time="2024-10-14T14:42:01+08:00" level=info msg="Allowed to access the referrers API, fallback if not supported"
time="2024-10-14T14:42:01+08:00" level=debug msg="Request #0\n> Request: \"HEAD\" \"https://testnotation.azurecr.io/v2/tsa/manifests/v1\"\n> Request headers:\n   \"Accept\": \"application/vnd.docker.distribution.manifest.v2+json, application/vnd.docker.distribution.manifest.list.v2+json, application/vnd.oci.image.manifest.v1+json, application/vnd.oci.image.index.v1+json, application/vnd.oci.artifact.manifest.v1+json\"\n   \"User-Agent\": \"notation/v1.2.0-beta.1+unreleased\""
time="2024-10-14T14:42:02+08:00" level=debug msg="Response #0\n< Response status: \"401 Unauthorized\"\n< Response headers:\n   \"Docker-Distribution-Api-Version\": \"registry/2.0\"\n   \"X-Content-Type-Options\": \"nosniff\"\n   \"X-Ms-Correlation-Request-Id\": \"a5874457-2d9f-4307-af69-4cf4dbf7de61\"\n   \"Content-Type\": \"application/json; charset=utf-8\"\n   \"Date\": \"Mon, 14 Oct 2024 06:42:04 GMT\"\n   \"Content-Length\": \"197\"\n   \"Connection\": \"keep-alive\"\n   \"Www-Authenticate\": \"Bearer realm=\\\"https://testnotation.azurecr.io/oauth2/token\\\",service=\\\"testnotation.azurecr.io\\\",scope=\\\"repository:tsa:pull\\\"\"\n   \"Server\": \"AzureContainerRegistry\"\n   \"Access-Control-Expose-Headers\": \"Docker-Content-Digest, WWW-Authenticate, Link, X-Ms-Correlation-Request-Id\"\n   \"Strict-Transport-Security\": \"max-age=31536000; includeSubDomains, max-age=31536000; includeSubDomains\""
time="2024-10-14T14:42:02+08:00" level=debug msg="started executing credential helper program docker-credential-desktop with action get"
time="2024-10-14T14:42:02+08:00" level=debug msg="successfully finished executing credential helper program docker-credential-desktop with action get"
time="2024-10-14T14:42:02+08:00" level=debug msg="Request #1\n> Request: \"GET\" \"https://testnotation.azurecr.io/oauth2/token?scope=repository%3Atsa%3Apull&service=testnotation.azurecr.io\"\n> Request headers:\n   \"Authorization\": \"*****\"\n   \"User-Agent\": \"notation/v1.2.0-beta.1+unreleased\""
time="2024-10-14T14:42:02+08:00" level=debug msg="Response #1\n< Response status: \"200 OK\"\n< Response headers:\n   \"X-Ms-Ratelimit-Remaining-Calls-Per-Second\": \"333.316667\"\n   \"Strict-Transport-Security\": \"max-age=31536000; includeSubDomains\"\n   \"Server\": \"AzureContainerRegistry\"\n   \"Date\": \"Mon, 14 Oct 2024 06:42:04 GMT\"\n   \"Content-Type\": \"application/json; charset=utf-8\"\n   \"Connection\": \"keep-alive\"\n   \"X-Ms-Correlation-Request-Id\": \"816d4eda-f1d4-4dbc-a547-31ba3d0c8309\""
time="2024-10-14T14:42:02+08:00" level=debug msg="Request #2\n> Request: \"HEAD\" \"https://testnotation.azurecr.io/v2/tsa/manifests/v1\"\n> Request headers:\n   \"Accept\": \"application/vnd.docker.distribution.manifest.v2+json, application/vnd.docker.distribution.manifest.list.v2+json, application/vnd.oci.image.manifest.v1+json, application/vnd.oci.image.index.v1+json, application/vnd.oci.artifact.manifest.v1+json\"\n   \"Authorization\": \"*****\"\n   \"User-Agent\": \"notation/v1.2.0-beta.1+unreleased\""
time="2024-10-14T14:42:03+08:00" level=debug msg="Response #2\n< Response status: \"200 OK\"\n< Response headers:\n   \"Access-Control-Expose-Headers\": \"Docker-Content-Digest, WWW-Authenticate, Link, X-Ms-Correlation-Request-Id\"\n   \"X-Ms-Request-Id\": \"e7fb1f0a-e886-4e70-a951-7278173f548f\"\n   \"Server\": \"AzureContainerRegistry\"\n   \"Content-Type\": \"application/vnd.docker.distribution.manifest.v2+json\"\n   \"Docker-Content-Digest\": \"sha256:c0669ef34cdc14332c0f1ab0c2c01acb91d96014b172f1a76f3a39e63d1f0bda\"\n   \"Etag\": \"\\\"sha256:c0669ef34cdc14332c0f1ab0c2c01acb91d96014b172f1a76f3a39e63d1f0bda\\\"\"\n   \"Date\": \"Mon, 14 Oct 2024 06:42:04 GMT\"\n   \"Connection\": \"keep-alive\"\n   \"Docker-Distribution-Api-Version\": \"registry/2.0\"\n   \"Strict-Transport-Security\": \"max-age=31536000; includeSubDomains, max-age=31536000; includeSubDomains\"\n   \"X-Content-Type-Options\": \"nosniff\"\n   \"X-Ms-Client-Request-Id\": \"\"\n   \"X-Ms-Correlation-Request-Id\": \"3a47014f-b8a1-48b2-b8b6-7a5ca43abb9b\"\n   \"Content-Length\": \"528\""

......
```

After setting the flag:
```
./notation.exe verify testnotation.azurecr.io/tsa:v1 -d
time=2024-10-14T14:44:38+08:00 level=info msg=Allowed to access the referrers API, fallback if not supported
time=2024-10-14T14:44:38+08:00 level=debug msg=Request #0
> Request: "HEAD" "https://testnotation.azurecr.io/v2/tsa/manifests/v1"
> Request headers:
   "Accept": "application/vnd.docker.distribution.manifest.v2+json, application/vnd.docker.distribution.manifest.list.v2+json, application/vnd.oci.image.manifest.v1+json, application/vnd.oci.image.index.v1+json, application/vnd.oci.artifact.manifest.v1+json"
   "User-Agent": "notation/v1.2.0-beta.1+unreleased"
time=2024-10-14T14:44:39+08:00 level=debug msg=Response #0
< Response status: "401 Unauthorized"
< Response headers:
   "Date": "Mon, 14 Oct 2024 06:44:41 GMT"
   "X-Content-Type-Options": "nosniff"
   "Strict-Transport-Security": "max-age=31536000; includeSubDomains, max-age=31536000; includeSubDomains"
   "X-Ms-Correlation-Request-Id": "d8dcba8f-33de-4fc3-ad0e-dd41fbe6c0cd"
   "Content-Type": "application/json; charset=utf-8"
   "Connection": "keep-alive"
   "Access-Control-Expose-Headers": "Docker-Content-Digest, WWW-Authenticate, Link, X-Ms-Correlation-Request-Id"
   "Server": "AzureContainerRegistry"
   "Docker-Distribution-Api-Version": "registry/2.0"
   "Content-Length": "197"
   "Www-Authenticate": "Bearer realm=\"https://testnotation.azurecr.io/oauth2/token\",service=\"testnotation.azurecr.io\",scope=\"repository:tsa:pull\""
time=2024-10-14T14:44:39+08:00 level=debug msg=started executing credential helper program docker-credential-desktop with action get
time=2024-10-14T14:44:39+08:00 level=debug msg=successfully finished executing credential helper program docker-credential-desktop with action get
time=2024-10-14T14:44:39+08:00 level=debug msg=Request #1
> Request: "GET" "https://testnotation.azurecr.io/oauth2/token?scope=repository%3Atsa%3Apull&service=testnotation.azurecr.io"
> Request headers:
   "Authorization": "*****"
   "User-Agent": "notation/v1.2.0-beta.1+unreleased"
time=2024-10-14T14:44:39+08:00 level=debug msg=Response #1
< Response status: "200 OK"
< Response headers:
   "Server": "AzureContainerRegistry"
   "Date": "Mon, 14 Oct 2024 06:44:41 GMT"
   "Content-Type": "application/json; charset=utf-8"
   "Connection": "keep-alive"
   "X-Ms-Correlation-Request-Id": "bd01ea37-20a3-4149-b0e6-beffe691675f"
   "X-Ms-Ratelimit-Remaining-Calls-Per-Second": "333.316667"
   "Strict-Transport-Security": "max-age=31536000; includeSubDomains"
time=2024-10-14T14:44:39+08:00 level=debug msg=Request #2
> Request: "HEAD" "https://testnotation.azurecr.io/v2/tsa/manifests/v1"
> Request headers:
   "Accept": "application/vnd.docker.distribution.manifest.v2+json, application/vnd.docker.distribution.manifest.list.v2+json, application/vnd.oci.image.manifest.v1+json, application/vnd.oci.image.index.v1+json, application/vnd.oci.artifact.manifest.v1+json"
   "Authorization": "*****"
   "User-Agent": "notation/v1.2.0-beta.1+unreleased"
```
@shizhMSFT

Reply:
This is also true for `ORAS`: https://github.com/oras-project/oras/blob/304a89fa124a9f2dc0296ccbd4bc0fe19a5c5e70/internal/trace/context.go#L41